DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -0,0 +1,216 @@
+# MVDR Beamforming
+This reference design demonstrates FPGA IO streaming in oneAPI. 
+
+***Documentation***:  
+* [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
+* [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
+* [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming. 
+ 
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04; Windows* 10
+| Hardware                          | Intel&reg; Programmable Acceleration Card (PAC) with Intel Arria&reg; 10 GX FPGA <br> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix&reg; 10 SX) <br> Intel Xeon&reg; CPU E5-1650 v2 @ 3.50GHz (host machine)
+| Software                          | Intel&reg; oneAPI DPC++ Compiler <br> Intel&reg; FPGA Add-On for oneAPI Base Toolkit
+| What you will learn               | How to perform IO streaming in oneAPI using the IO pipes SYCL extension.
+| Time to complete                  | 1 hour
+
```

Comment:
Performance wasn't the main goal of this reference design, it's main purpose is to demonstrate a realistic system designed for IO streaming.  The design has not been fmax-optimized to the level of most of our reference designs, and as such I'm not sure the performance is sufficiently impressive to justify documenting here.

Reply:
This makes this design somewhat different than the rest of the FPGA Reference Designs, all of which showcase competitive performance in their respective application. This is a bit of an odd duck, as it shares characteristics of Reference Designs and of Design Patterns.

Is it your intention to further optimize the performance of this design in the future?

Please consider whether it would be worthwhile including a "Performance" section here as in the other designs, and stating what you said here.